pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -317,15 +320,20 @@ async def run(self):
                 # update jira promote task status
                 self._logger.info("Updating promote release subtask")
                 jira_issue_key = group_config.get("release_jira")
-                if jira_issue_key:
+                if jira_issue_key and not self.dry_run:
                     parent_jira = self._jira_client.get_issue(jira_issue_key)
-                    subtask = self._jira_client.get_issue(parent_jira.fields.subtasks[3].key)
-                    self._jira_client.add_comment(
-                        subtask,
-                        "promote release job : {}".format(os.environ.get("BUILD_URL"))
-                    )
-                    self._jira_client.assign_to_me(subtask)
-                    self._jira_client.close_task(subtask)
+                    title = "[Wed] Promote the tested nightly"
```

Comment:
Shouldn't it be `[Thu]`, as we're preparing-promoting on Thursdays now? If so, there's probably other places where this should change

Reply:
Ah my bad this is a different subtask, I've updated it now https://github.com/openshift-eng/art-tools/pull/52/commits/8338a7e9b3721e8b0a489424fd6cf91e63539b94